### ISSUE TITLE
Option to set key expiration on generate

### ISSUE THREAD

**Issue by MoonlightCapital** [lucanaef/PGPro#59]
There should be an option to generate a pair of key with a set expiration, instead of making them always never expire.

**Comment by lucanaef** [comment]
I agree! I have not implemented this yet, since the cryptography framework I'm currently using has no option to set key expiration dates.
